stock_scanner_web/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
```

Comment:
Please put this in [README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)